src/Board.php

```diff
@@ -149,7 +149,7 @@ public function rewind(): ?Piece
 
     public function jsonSerialize(): string
     {
-        return \json_encode($this->squares);
+        return \implode(' ', $this->squares);
```

Comment:
you should probably change the name of the method, because it will do something else now

Reply:
We can't change method name, it's needed to be able to call `json_encode()` on the object.

Moreover, it's perfectly fine to return a string in `jsonSerialize()`, that just need to return a "JSON representation of a value" (according to official php documentation)

Reply:
I am aware of this, but should this return string be a valid json string?

Reply:
It's valid. This is an example:
`"r n b q k b n r p p p p p p p p                     P            P P P P  P P P R N B Q K B N R"`

I validated it on https://jsonlint.com/

Reply:
actually, just like the string itself, ok you convinced me :wink: